lib/invest/invest-app.ts

```diff
@@ -3,15 +3,45 @@ import { investService } from './invest-service';
 
 export const investApp = express();
 
+investApp.get('/snipe-player', async (req, res) => {
```

Comment:
Sorry, I'll not accept this one on invest jobs. We have exact same in auto-buyer. 
And for invest, the plan is to pass a futbin link for set of players. 
It's not sniping but it's reducing scope to more popular players.